addons/mail/static/src/models/call_participant_card_popover_content_view.js

```diff
@@ -0,0 +1,84 @@
+/** @odoo-module **/
+
+import { registerModel } from '@mail/model/model_core';
+import { attr, one } from '@mail/model/model_field';
+
+import { sprintf } from '@web/core/utils/strings';
+
+registerModel({
+    name: 'CallParticipantCardPopoverContentView',
+    recordMethods: {
+        /**
+         * @param {Event} ev
+         */
+        onChangeVolume(ev) {
+            this.callParticipantCard.rtcSession && this.callParticipantCard.rtcSession.setVolume(parseFloat(ev.target.value));
+        },
+        /**
+         * @private
+         * @returns {boolean}
+         */
+        _computeHasConnectionInfo() {
+            return Boolean(this.callParticipantCard.rtcSession && this.env.debug && this.callParticipantCard.channelMember.channel.thread.rtc);
+        },
+        /**
+         * @private
+         * @returns {string}
+         */
+        _computeInboundConnectionTypeText() {
+            if (!this.callParticipantCard.rtcSession || !this.callParticipantCard.rtcSession.remoteCandidateType) {
+                return sprintf(this.env._t('From %s: no connection'), this.callParticipantCard.channelMember.persona.name);
```

Comment:
format:
```js
if (
   A ||
   B
) {
```

Reply:
you should zoom out a little bit, it happens on github sometimes....